<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textInformation">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>271</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>101</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="labe_name_2">
       <property name="text">
        <string>aa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_age_2">
       <property name="text">
        <string>14</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_sex_2">
       <property name="text">
        <string>man</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_height_2">
       <property name="text">
        <string>120</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>101</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labe_name">
       <property name="text">
        <string>name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_age">
       <property name="text">
        <string>age: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_sex">
       <property name="text">
        <string>sex: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_height">
       <property name="text">
        <string>Height(cm): </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_infor">
       <property name="text">
        <string>Information: </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>81</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonName">
       <property name="text">
        <string>EditName</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAge">
       <property name="text">
        <string>EditAge</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSex">
       <property name="text">
        <string>EditSex</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonHeight">
       <property name="text">
        <string>EditHeight</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonInfo">
       <property name="text">
        <string>Edit Info</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
